### PR TITLE
*: update basic features for 7.0

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

as title.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/13047
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,212 +20,212 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| 数据导入和导出 | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
+| [数据迁移工具](/migration-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiCDC 支持保存数据到存储服务 (S3/NFS/Azure Blob Storage)](/ticdc/ticdc-sink-to-cloud-storage.md) | E | E | E | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:195]: @Frank945946 PTAL
```suggestion
| [TiCDC 支持保存数据到存储服务 (S3/NFS/Azure Blob Storage/GCP)](/ticdc/ticdc-sink-to-cloud-storage.md) | Y | E | E | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 2]
```diff
@@ -20,212 +20,212 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [Optimizer hints](/optimizer-hints.md)                              |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | Y        |
-| [MPP 执行引擎](/explain-mpp.md)                                     |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | N        |
-| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) |  Y | N  | N | N | N | N | N | N | N |
-| [索引合并](/explain-index-merge.md)                                 |    Y    |    Y    |    Y    |    Y     | E | E | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md)                |    Y    |    Y    |    Y    | E | E |    N     |    N     |    N     | N        |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E |
+| 高级 SQL 功能 | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|---|:---:|
+| [执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y | Y | Y | Y | E | E | E | E |
```
Comment (Oreoxmt) [inline basic-features.md:92]:
```suggestion
| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y | Y | Y | Y | E | E | E | E |
| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) | E | N | N | N | N | N | N | N | N | N |
```
@qw4990 @easonn7 PTAL

File: basic-features.md [hunk 3]
```diff
@@ -20,212 +20,212 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
+| [数据迁移工具](/migration-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiCDC 支持保存数据到存储服务 (S3/NFS/Azure Blob Storage/GCP)](/ticdc/ticdc-sink-to-cloud-storage.md) | Y | E | E | N | N | N | N | N | N | N |
+| [TiCDC 支持在两个 TiDB 集群之间进行双向复制](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | Y | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:197]: @Frank945946 PTAL
```suggestion
| [TiCDC OpenAPI v2](/ticdc/ticdc-open-api-v2.md) | Y | N | N | N | N | N | N | N | N | N |

```

File: basic-features.md [hunk 4]
```diff
@@ -20,212 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [List 分区](/partitioned-table.md)                   | Y | Y | Y | E | E | E | E | E |    N    |
-| [List COLUMNS 分区](/partitioned-table.md)           | Y | Y | Y | E | E | E | E | E |    N    |
-| [`EXCHANGE PARTITION`](/partitioned-table.md)               | Y | Y  | E | E | E | E | E | E |    N    |
-| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | E | E | E | E |      N       |    N    |
-| [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) | Y | Y | N | N | N |N | N | N |      N       |
-| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | E | E | N | N | N |N | N | N |      N       |
+| 分区 | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|---|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Range 分区](/partitioned-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Hash 分区](/partitioned-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (qiancai) [inline basic-features.md:143]:
```suggestion
| [Hash 分区](/partitioned-table.md#hash-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
| [Key 分区](/partitioned-table.md#key-分区) | Y | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 5]
```diff
@@ -20,212 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [Hash 分区](/partitioned-table.md)                   | Y | Y  | Y | Y            |      Y       |      Y       |      Y       |      Y       |    Y    |
-| [List 分区](/partitioned-table.md)                   | Y | Y | Y | E | E | E | E | E |    N    |
-| [List COLUMNS 分区](/partitioned-table.md)           | Y | Y | Y | E | E | E | E | E |    N    |
-| [`EXCHANGE PARTITION`](/partitioned-table.md)               | Y | Y  | E | E | E | E | E | E |    N    |
-| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | E | E | E | E |      N       |    N    |
-| [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) | Y | Y | N | N | N |N | N | N |      N       |
-| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | E | E | N | N | N |N | N | N |      N       |
+| 分区 | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|---|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Range 分区](/partitioned-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (qiancai) [inline basic-features.md:142]:
```suggestion
| [Range 分区](/partitioned-table.md#range-分区) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```

File: basic-features.md [hunk 6]
```diff
@@ -20,212 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Hash 分区](/partitioned-table.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [List 分区](/partitioned-table.md) | Y | Y | Y | Y | E | E | E | E | E | N |
+| [List COLUMNS 分区](/partitioned-table.md) | Y | Y | Y | Y | E | E | E | E | E | N |
```
Comment (qiancai) [inline basic-features.md:145]:
```suggestion
| [List 分区](/partitioned-table.md#list-分区) | Y | Y | Y | Y | E | E | E | E | E | N |
| [List COLUMNS 分区](/partitioned-table.md#list-columns-分区) | Y | Y | Y | Y | E | E | E | E | E | N |
```

File: basic-features.md [hunk 7]
```diff
@@ -20,212 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [`EXCHANGE PARTITION`](/partitioned-table.md) | Y | Y | Y | E | E | E | E | E | E | N |
```
Comment (qiancai) [inline basic-features.md:146]:
```suggestion
| [`EXCHANGE PARTITION`](/partitioned-table.md) | Y | Y | Y | E | E | E | E | E | E | N |
| [`REORGANIZE PARTITION`](/partitioned-table.md#重组分区) | Y | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 8]
```diff
@@ -20,212 +20,213 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [MPP 执行引擎](/explain-mpp.md)                                     |    Y    |    Y    |    Y    |    Y     |    Y     |    Y     |    Y     |    Y     | N        |
-| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) |  Y | N  | N | N | N | N | N | N | N |
-| [索引合并](/explain-index-merge.md)                                 |    Y    |    Y    |    Y    |    Y     | E | E | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md)                |    Y    |    Y    |    Y    | E | E |    N     |    N     |    N     | N        |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E |
+| 高级 SQL 功能 | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|---|:---:|
+| [执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y | Y | Y | Y | E | E | E | E |
+| [执行计划管理 (SPM)](/sql-plan-management.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定) | E | E | E | N | N | N | N | N | N | N |
```
Comment (songrijie) [inline basic-features.md:94]: 这个已经GA了

Comment (songrijie) [reply basic-features.md:94]:
```suggestion
| [根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定) | Y | Y | E | N | N | N | N | N | N | N |
```

**Review by Frank945946 — APPROVED** [review]

**Comment by songrijie** [comment]
Rest LGTM

File: basic-features.md [hunk 9]
```diff
@@ -20,212 +20,214 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [复合或非整型主键上的聚簇索引](/constraints.md)                        |  Y  |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
-| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | E | N | N | N | N | N | N | N |
-| [外键约束](/constraints.md#外键约束) | Y | Y | N | N | N | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-add-index.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [复合主键](/constraints.md#主键约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [唯一约束](/constraints.md#唯一约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | E | N | N | N | N | N | N | N |  |
+| [外键约束](/constraints.md#外键约束) | Y | Y | N | N | N | N | N | N | N |  |
```
Comment (joey-yez) [inline basic-features.md:63]: | [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | E | N | N | N | N | N | N | N | N | N |

File: basic-features.md [hunk 10]
```diff
@@ -20,212 +20,214 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
```
Comment (joey-yez) [inline basic-features.md:62]:
```suggestion
| [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | E | N | N | N | N | N | N | N | N | N |

```

File: basic-features.md (hunk 9 above)
Comment (joey-yez) [inline basic-features.md:63]:
```suggestion
| [TiFlash 延迟物化](/tiflash/tiflash-late-materialization.md) | E | N | N | N | N | N | N | N | N | N |

```

File: basic-features.md [hunk 11]
```diff
@@ -20,212 +20,214 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [`SET CONFIG`](/dynamic-config.md) | Y | Y | Y | Y | E | E | E | E | E | E |
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | E | N | N | N | N | N | N |
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | Y | E | N | N | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | Y | E | E | N | N | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore) | Y | Y | Y | N | N | N | N | N | N | N |
+| [PITR](/br/br-pitr-guide.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值) | Y | Y | Y | N | N | N | N | N | N | N |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | E | N | N | N | N | N | N | N | N |
```
Comment (joey-yez) [inline basic-features.md:231]:
```suggestion
| [TiFlash 存算分离架构与 S3 支持](/tiflash/tiflash-disaggregated-and-s3.md) | E | N | N | N | N | N | N | N | N | N |

```

File: basic-features.md [hunk 12]
```diff
@@ -20,212 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [复合或非整型主键上的聚簇索引](/constraints.md)                        |  Y  |  Y  |  Y   |  Y    |    Y     |    Y     |    Y     |    Y     |    Y     |    N     |
-| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | E | N | N | N | N | N | N | N |
-| [外键约束](/constraints.md#外键约束) | Y | Y | N | N | N | N | N | N | N |
+| [不可见索引](/sql-statements/sql-statement-add-index.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [复合主键](/constraints.md#主键约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [唯一约束](/constraints.md#唯一约束) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [复合或非整型主键上的聚簇索引](/constraints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | E | N | N | N | N | N | N | N |  |
+| [外键约束](/constraints.md#外键约束) | Y | Y | N | N | N | N | N | N | N |  |
```
Comment (ran-huang) [inline basic-features.md:62]:
```suggestion
| [多值索引](/sql-statements/sql-statement-create-index.md#多值索引) | E | E | N | N | N | N | N | N | N | N |
| [外键约束](/constraints.md#外键约束) | Y | Y | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 13]
```diff
@@ -20,212 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [`ALTER TABLE ... COMPACT`](/sql-statements/sql-statement-alter-table-compact.md) | Y | Y | E | N | N | N | N | N | N |
-| [表级锁 (Table Lock)](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) | E | E | E | E | E | E | E | E | E |
-| [物化列式存储的查询结果](/tiflash/tiflash-results-materialization.md) | E | E | N | N | N | N | N | N | N |
+| SQL 语句 [^3] | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| `SELECT`，`INSERT`，`UPDATE`，`DELETE`，`REPLACE` | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| `INSERT ON DUPLICATE KEY UPDATE` | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| `LOAD DATA INFILE` | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| `SELECT INTO OUTFILE` | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| `INNER JOIN`, LEFT\|RIGHT [OUTER] JOIN | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```
Comment (ran-huang) [inline basic-features.md:73]:
```suggestion
| `INNER JOIN`, <code>LEFT\|RIGHT [OUTER] JOIN</code> | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
```

File: basic-features.md [hunk 14]
```diff
@@ -20,212 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
+| [数据迁移工具](/migration-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Change data capture (CDC)](/ticdc/ticdc-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiCDC 支持保存数据到存储服务 (S3/NFS/Azure Blob Storage/GCP)](/ticdc/ticdc-sink-to-cloud-storage.md) | Y | E | E | N | N | N | N | N | N | N |
+| [TiCDC 支持在两个 TiDB 集群之间进行双向复制](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| [TiCDC OpenAPI v2](/ticdc/ticdc-open-api-v2.md) | Y | N | N | N | N | N | N | N | N | N |
+| [TiCDC OpenAPI v2](/ticdc/ticdc-open-api-v2.md) | Y | N | N | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:201]:
```suggestion
```

File: basic-features.md [hunk 15]
```diff
@@ -20,212 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [TiUP 部署](/tiup/tiup-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Kubernetes operator](https://docs.pingcap.com/tidb-in-kubernetes/stable) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [内置物理备份](/br/backup-and-restore-use-cases.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Global Kill](/sql-statements/sql-statement-kill.md) | Y | Y | Y | Y | E | E | E | E | E | E |
+| [Lock View](/information-schema/information-schema-data-lock-waits.md) | Y | Y | Y | Y | Y | Y | Y | E | E | E |
+| [`SHOW CONFIG`](/sql-statements/sql-statement-show-config.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [`SET CONFIG`](/dynamic-config.md) | Y | Y | Y | Y | E | E | E | E | E | E |
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | E | N | N | N | N | N | N |
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | Y | E | N | N | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | Y | E | E | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:227]:
```suggestion
| [后台限流](/tikv-configuration-file.md#后台限流) | E | E | E | N | N | N | N | N | N | N |
```

**Comment by easonn7** [comment]
LGTM

**Review by Oreoxmt — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 16]
```diff
@@ -20,212 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [`SET CONFIG`](/dynamic-config.md) | Y | Y | Y | Y | E | E | E | E | E | E |
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | E | N | N | N | N | N | N |
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | Y | E | N | N | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | E | E | E | N | N | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore) | Y | Y | Y | N | N | N | N | N | N | N |
+| [PITR](/br/br-pitr-guide.md) | Y | Y | Y | N | N | N | N | N | N | N |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值) | Y | Y | Y | N | N | N | N | N | N | N |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | E | N | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:233]:
```suggestion
| [资源管控 (Resource Control)](/tidb-resource-control.md) | E | E | N | N | N | N | N | N | N | N |
```

File: basic-features.md [hunk 17]
```diff
@@ -20,212 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                     |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
-| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                   |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
-| [TiCDC 支持保存数据到存储服务 (S3/NFS/Azure Blob Storage)](/ticdc/ticdc-sink-to-cloud-storage.md)  |   E   |   E   |   N    |   N    |   N    |   N    |   N    |   N    |   N    |
-| [TiCDC 支持在两个 TiDB 集群之间进行双向复制](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | N | N | N | N | N | N | N |
+| 数据导入和导出 | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
```
Comment (hfxsd) [inline basic-features.md:195]:
```suggestion
| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | E | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
```

File: basic-features.md [hunk 18]
```diff
@@ -20,212 +20,218 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)                     |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
-| [Change data capture (CDC)](/ticdc/ticdc-overview.md)                   |   Y   |   Y   |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |   Y    |
-| [TiCDC 支持保存数据到存储服务 (S3/NFS/Azure Blob Storage)](/ticdc/ticdc-sink-to-cloud-storage.md)  |   E   |   E   |   N    |   N    |   N    |   N    |   N    |   N    |   N    |
-| [TiCDC 支持在两个 TiDB 集群之间进行双向复制](/ticdc/ticdc-bidirectional-replication.md) | Y | Y | N | N | N | N | N | N | N |
+| 数据导入和导出 | 7.0 | 6.6 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [快速导入 (TiDB Lightning)](/tidb-lightning/tidb-lightning-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| mydumper 逻辑导出 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 | 已废弃 |
+| [Dumpling 逻辑导出](/dumpling-overview.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | E | Y | Y | Y | Y | Y | Y | Y | Y | N [^5] |
```
Comment (hfxsd) [inline basic-features.md:195]:
```suggestion
| [事务 `LOAD DATA`](/sql-statements/sql-statement-load-data.md) | E [^5] | Y | Y | Y | Y | Y | Y | Y | Y | N [^6] |
```

File: basic-features.md [hunk 19]
```diff
@@ -235,4 +241,6 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
 
 [^4]: 从 [TiDB v6.4.0](/releases/release-6.4.0.md) 开始，支持[高性能、全局单调递增的 `AUTO_INCREMENT` 列](/auto-increment.md#mysql-兼容模式)
 
-[^5]: 对于 TiDB v4.0，事务 `LOAD DATA` 不保证原子性。
+[^5]: 从 [TiDB v7.0.0](/releases/release-7.0.0.md) 开始，新增参数 `FORMAT`、`FIELDS DEFINED NULL BY`、`With batch_size=<number>,detached`，以及新增支持从 S3 和 GCS 导入数据，均为实验特性。
```
Comment (ran-huang) [inline basic-features.md:244]:
```suggestion
[^5]: 对于 [TiDB v7.0.0](/releases/release-7.0.0.md)，新增参数 `FORMAT`、`FIELDS DEFINED NULL BY`、`With batch_size=<number>,detached`，以及新增支持从 S3 和 GCS 导入数据，均为实验特性。
```

**Pull request merged**